src/Bridge/ClientRepository.php

```diff
@@ -15,12 +15,18 @@ class ClientRepository implements ClientRepositoryInterface
      */
     protected ClientModelRepository $clients;
 
+    /**
+     * The hasher implementation.
+     */
+    protected Hasher $hasher;
+
     /**
      * Create a new repository instance.
      */
-    public function __construct(ClientModelRepository $clients)
+    public function __construct(ClientModelRepository $clients, Hasher $hasher)
```

Comment:
what about https://stitcher.io/blog/constructor-promotion-in-php-8 ?

Reply:
I prefer to follow the same style of the file I'm changing.